CMakeLists.txt

```diff
@@ -42,24 +45,18 @@ target_include_directories(usgscsm
 )
 
 # Setup for GoogleTest
-
 find_package (Threads)
 
 target_link_libraries(usgscsm
                       ${CSM_LIBRARY}
                       gtest ${CMAKE_THREAD_LIBS_INIT})
 
 if(WIN32)
-  option(CMAKE_USE_WIN32_THREADS_INIT "using WIN32 threads" ON)
-  option(gtest_disable_pthreads "Disable uses of pthreads in gtest." ON)
-  install(TARGETS usgscsm
-    RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR})
-    install(DIRECTORY ${USGSCSM_INCLUDE_DIRS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+  install(TARGETS usgscsm RUNTIME DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
This should remove the duplication. I am not 100% sure why RUNTIME is needed on win. I found this in a solution, so we should not consider this to be a best practice.